.github/workflows/conformance-clustermesh.yaml

```diff
@@ -0,0 +1,504 @@
+        fi
+
+        # Skip node-port tests if kube-proxy-replacement is disabled, as not working (#23266)
+        # TODO: Remove once cilium/cilium-cli#1401 gets merged and released.
+        if [[ "${{ matrix.kube-proxy }}" == "iptables" ]]; then
+          CONNECTIVITY_TEST_DEFAULTS="$CONNECTIVITY_TEST_DEFAULTS \
+            --test='!/pod-to-.*-nodeport'"
+        fi
+
+        echo cilium_install_defaults="${CILIUM_INSTALL_DEFAULTS} ${CILIUM_INSTALL_TUNNEL} ${CILIUM_INSTALL_IPFAMILY}" >> $GITHUB_OUTPUT
+        echo hubble_enable_defaults=${HUBBLE_ENABLE_DEFAULTS} >> $GITHUB_OUTPUT
+        echo connectivity_test_defaults=${CONNECTIVITY_TEST_DEFAULTS} >> $GITHUB_OUTPUT
+        echo clustermesh_enable_defaults=${CLUSTERMESH_ENABLE_DEFAULTS} >> $GITHUB_OUTPUT
+
+        echo kind_pod_cidr_1=${KIND_POD_CIDR_1} >> $GITHUB_OUTPUT
+        echo kind_svc_cidr_1=${KIND_SVC_CIDR_1} >> $GITHUB_OUTPUT
+        echo kind_pod_cidr_2=${KIND_POD_CIDR_2} >> $GITHUB_OUTPUT
+        echo kind_svc_cidr_2=${KIND_SVC_CIDR_2} >> $GITHUB_OUTPUT
+
+        echo sha=${SHA} >> $GITHUB_OUTPUT
+        echo owner=${OWNER} >> $GITHUB_OUTPUT
+
+    - name: Set commit status to pending
+      uses: Sibz/github-status-action@650dd1a882a76dbbbc4576fb5974b8d22f29847f # v1.1.6
+      with:
+        authToken: ${{ secrets.GITHUB_TOKEN }}
+        sha: ${{ steps.vars.outputs.sha }}
+        context: ${{ github.workflow }}
+        description: Connectivity test in progress...
+        state: pending
+        target_url: ${{ env.check_url }}
+
+    - name: Checkout code
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      with:
+        ref: ${{ steps.vars.outputs.sha }}
+        persist-credentials: false
+
+    - name: Install Cilium CLI
+      run: |
+        curl -sSL --remote-name-all https://github.com/cilium/cilium-cli/releases/download/${{ env.cilium_cli_version }}/cilium-linux-amd64.tar.gz{,.sha256sum}
+        sha256sum --check cilium-linux-amd64.tar.gz.sha256sum
+        sudo tar xzvfC cilium-linux-amd64.tar.gz /usr/local/bin
+        rm cilium-linux-amd64.tar.gz{,.sha256sum}
+        cilium version
+
+    - name: Generate Kind configuration files
+      run: |
+        PODCIDR=${{ steps.vars.outputs.kind_pod_cidr_1 }} \
+          SVCCIDR=${{ steps.vars.outputs.kind_svc_cidr_1 }} \
+          IPFAMILY=${{ matrix.ipFamily }} \
+          KUBEPROXYMODE=${{ matrix.kube-proxy }} \
+          envsubst < ./.github/kind-config.yaml.tmpl > ./.github/kind-config-cluster1.yaml
+
+        PODCIDR=${{ steps.vars.outputs.kind_pod_cidr_2 }} \
+          SVCCIDR=${{ steps.vars.outputs.kind_svc_cidr_2 }} \
+          IPFAMILY=${{ matrix.ipFamily }} \
+          KUBEPROXYMODE=${{ matrix.kube-proxy }} \
+          envsubst < ./.github/kind-config.yaml.tmpl > ./.github/kind-config-cluster2.yaml
+
+    - name: Create Kind cluster 1
+      uses: helm/kind-action@d8ccf8fb623ce1bb360ae2f45f323d9d5c5e9f00
```

Comment:
please add a comment with the version of this commit so that renovate can update this dependency automatically. Similar to what is done on line 330

Reply:
Missed that, sorry